article/_posts/2021-09-02-sindyautoencoder.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: review
+title: "Data-driven discovery of coordinates and governing equations"
+tags: dimensionality-reduction autoencoder
+author: "Thierry Judge"
+cite:
+    authors: "Kathleen Champion, Bethany Lusch, J. Nathan Kutz1, Steven L. Brunton"
```

Comment:
The `1` at the end of `Kutz` seems to be a typo :stuck_out_tongue:

Reply:
Fixed in cdcfddf

Reply:
Fixed in cdcfddf